src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER;
+
+    static {
```

Comment:
It's redundant to use static block in this case. You can initialise DateTimeFormatter when you declare this variable.
Also, you correct format of data dd.MM.yyyy